src/pl.x64.json

```diff
@@ -11,7 +11,8 @@
 			"repository": "https://github.com/jcaillon/3P/releases/download/v1.8.4/3P_x64.zip",
 			"description": "[Requires .net framework 4.6.1]\r\n3P is a notepad++ plug-in designed to help writing OpenEdge ABL (formerly known as Progress 4GL) code. It provides :\r\n- a powerful auto-completion\r\n- tool-tips on every words\r\n- a code explorer to quickly navigate through your code\r\n- a file explorer to easily access all your sources\r\n- the ability to run/compile and even PROLINT your source file with an in-line visualization of errors\r\n- more than 50 options to better suit your needs\r\n- and so much more!\r\nVisit http://jcaillon.github.io/3P/ for more details on the plugin",
 			"author": "Julien Caillon",
-			"homepage": "https://jcaillon.github.io/3P/"
+			"homepage": "https://jcaillon.github.io/3P/",
+			"release-date": "2019-02-23"
```

Comment:
Have you tested also the existing implementation of N++7.7 how it behaves with this additional field value? Is it just ignored?

Reply:
Yes. I've test with N++ 7.7, the new field is just ignored.